src/generators/operationGenerator.ts

```diff
@@ -579,7 +581,7 @@ function getOperationParameterSignatures(
 function findLastRequiredParamIndex(
   params: ParameterWithDescription[]
 ): number {
-  for (let i = params.length; i--;) {
+  for (let i = params.length; i--; ) {
```

Comment:
Would it be possible to revert formatting changes like this one?

Reply:
Will be careful next time.